app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -82,29 +83,22 @@ public BridgeConfig bridgeConfig() {
     }
 
     @Bean(name = "jedisPool")
-    public JedisPool jedisPool(final BridgeConfig config) {
-
+    public JedisPool jedisPool(final BridgeConfig config) throws Exception {
         // Configure pool
         final JedisPoolConfig poolConfig = new JedisPoolConfig();
         poolConfig.setMaxTotal(config.getPropertyAsInt("redis.max.total"));
```

Comment:
Definitely. Now passing it to the JedisPool constructor, and also changed logging NOT to log the entire URL we're getting from Heroku.